Flicker/Global/Utils/FirebaseManager.swift

```diff
@@ -70,11 +70,6 @@ final class FirebaseManager: NSObject {
             
             try await firestore.collection("users").document(uid).setData(userData)
         } catch {
-//            print(email)
-//            print(name)
-//            print(uid)
-//            print(profileImage)
-//            print(error)
             print("Store User error")
```

Comment:
p3

요 부분 error 프린트할 수 있게 바꿔줍시다!
```suggestion
            print(error)
```